man/coap_session.txt.in

```diff
@@ -0,0 +1,386 @@
+// -*- mode:doc; -*-
+// vim: set syntax=asciidoc,tw=0:
+
+coap_session(3)
+=================
+:doctype: manpage
+:man source:   coap_context
+:man version:  @PACKAGE_VERSION@
+:man manual:   libcoap Manual
+
+NAME
+----
+coap_session,
+coap_new_client_session,
+coap_new_client_session_psk,
+coap_new_client_session_pki,
+coap_session_reference,
+coap_session_release,
+coap_session_set_mtu,
+coap_session_max_pdu_size,
+coap_session_str - work with CoAP contexts
```

Comment:
"work with CoAP contexts" is a bit misleading here as these functions primarily deal with sessions.

Reply:
Corrected to "work with CoAP sessions"